README.md

```diff
@@ -1,13 +1,19 @@
-# Project Todos
+# Project Todos ✓
+This project was all about creating a Todo-list in React, using Redux, creating our own reducers, and implementing all functions that are relevant for a Todo-app.
 
-Replace this readme with your own information about your project.
+## The problem
+I started off by sketching a mockup with pen and paper to figure out what functions I wanted, what components I would need and an overall basic take on layout. Heading into VSCode, I created a pretend todo-list to use while constructing the app. Having built all basic html construction and very basic styling with Styled Components, I gave the Redux dispatch/actions a go. After managing to get "Add new task"-function to work it felt like the rest of the project went by really smooth.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Turns out I really like working with Redux, and even though many things were new to me, I feel like I was able to try all sorts of things out. By the end of the project I could focus more on zooming in on each action, making sure I understand exatcly what I have done and why. Immutability is still a bit fuzzy for me, though. I found this project to be very valuable regarding handling arrays, using map, filter and find a lot.
 
-## The problem
+If I'd have more time, I would like to look into how to work with custom alerts, reminders and setting deadlines using dates, also being able to export/import tasks from/to desktop/mobile etc. Maybe it would have been nice practice to let the user add its own categories, and by doing so also updating the filter by category-functions.
```

Comment:
Karin, you should be SO proud, don't have really much more to say than you have done an amazing job! Your code is clean and simple to follow, you have stretched the goals to black level and styled it so so nice 👏 🏅 😄 

Very well done! Continue like this 👍 (I am gonna use this app) ❤️